devel/ansible/roles/dev/files/.bashrc

```diff
@@ -9,6 +9,7 @@ fi
 # export SYSTEMD_PAGER=
 
 shopt -s expand_aliases
+alias bci="sudo dnf install -y docker parallel && sudo systemctl enable docker && sudo systemctl start docker && sudo devel/run_tests.sh $@"
```

Comment:
do we really want to run dnf install docker and systemctl everytime this command is used ? Why not just add these to the ansible role and make `btest` be the same as the CI ?

Reply:
I thought about that, but most installs probably won't use this command and the SELinux bits of the install take an annoyingly long time. That said, I'm happy to change it if you prefer because I do feel "on the fence" about it.

Also, weirdly, I will never use this command because it causes one of my computers to freeze, and the command causes the Vagrant VM to irreversibly pause on my other computer. I've not been able to figure out why this happens on either computer, but @jeremycline says that doing this on his system works so maybe it'll work for some people.

Reply:
I've thought about it some more and I think I am swayed. I kinda felt bad about doing it this way in the first place.

I'll close for now and reopen later.